java/client/src/org/openqa/selenium/support/ui/CheckedFunction.java

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright 2007-2015 Selenium committers
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+ */
+
+package org.openqa.selenium.support.ui;
+
+/**
+ * Single argument predicate allowed to throw checked exceptions.
```

Comment:
This is only a predicate in the context of `FluentWait`. Change to

    A function that transforms an input value into another.